static/src/stylesheets/module/content-garnett/_gallery-lightbox.scss

```diff
@@ -14,6 +14,7 @@ $gallery-lightbox-sidebar-breakpoint: desktop;
     background-color: lighten($brightness-7, 2.5%);
     overflow: hidden;
     z-index: $zindex-lightbox;
+    box-sizing: border-box;
```

Comment:
We add this because it seems like some browsers which don't support `dialog` at the box aren't applying this - meaning that lightbox was extending off the edge of the window